assets/js/blocks/mini-cart/mini-cart-contents/inner-blocks/filled-mini-cart-contents-block/frontend.tsx

```diff
@@ -12,13 +18,33 @@ const FilledMiniCartContentsBlock = ( {
 	children,
 	className,
 }: FilledMiniCartContentsBlockProps ): JSX.Element | null => {
-	const { cartItems } = useStoreCart();
+	const { cartItems, cartItemErrors } = useStoreCart();
+
+	const { createErrorNotice } = useDispatch( 'core/notices' );
+
+	// Ensures any cart errors listed in the API response get shown.
+	useEffect( () => {
+		cartItemErrors.forEach( ( error ) => {
+			createErrorNotice( decodeEntities( error.message ), {
+				isDismissible: true,
+				id: error.code,
+				context: 'wc/cart',
```

Comment:
Hi @gigitux, I am just curious. Isn't it possible to hide the X button using `isDismissible: false,` property instead of using CSS? i.e.
```jsx
createErrorNotice( decodeEntities( error.message ), {
	isDismissible: false,
	id: error.code,
	context: 'wc/cart',
} );
```

Reply:
Good catch! I fixed the code with your suggestion. Also, I removed the not necessary SCSS 👍

Reply:
Awesome! Thanks 🚀